webpack.config.js

```diff
@@ -22,7 +22,8 @@ module.exports = {
   entry: {
     app: './js/index.app.js',
     dashboard_music_widget: './js/index.dashboard.js',
-    files_music_player: './js/index.embedded.js'
+    files_music_player: './js/index.embedded.js',
+    scrobble_getsession_result: './js/index.getsessionresult.js'
```

Comment:
There's no file index.getsessionresult.js included in the PR. Is this unnecessary or has the file been forgotten?